verilog/tools/formatter/verilog_format.cc

```diff
@@ -179,24 +182,22 @@ bool formatOneFile(absl::string_view filename,
       std::cout << content;
     }
     // Print the error message last so it shows up in user's console.
-    std::cerr << format_status.message() << std::endl;
+    std::cerr << filename << format_status.message() << " ";
     switch (format_status.code()) {
       case StatusCode::kCancelled:
       case StatusCode::kInvalidArgument:
         break;
       case StatusCode::kDataLoss:
-        std::cerr << "Problematic formatter output is:\n"
-                  << formatted_output << "<<EOF>>" << std::endl;
+        std::cerr << "\nProblematic formatter output is:\n"
+                  << formatted_output << "<<EOF>>";
         break;
       default:
-        std::cerr << "[other error status]" << std::endl;
+        std::cerr << "[other error status]";
         break;
     }
-    if (absl::GetFlag(FLAGS_failsafe_success)) {
-      // original text was preserved, and --inplace modification is skipped.
-      return true;
-    }
-    return false;
+    std::cerr << std::endl;
```

Comment:
but then the accumulation message `[other error status]` would be on its own line.

If we don't make it a continuation, we have tp put the format_status.message() print on every switch/case instead of above.

Reply:
Ok, repeating the message now which allows for full newline printing in each of them. Introduced `ostream& FileMsg(filename)` to reduce and unify boilerplate for each message that prints filenames with correct `<filename>:<space>` spacing.